sysext.just

```diff
@@ -304,14 +304,14 @@ reset-selinux-labels target:
 
     filecontexts="/etc/selinux/targeted/contexts/files/file_contexts"
     echo "🏷️ Resetting SELinux labels"
-    podman run --rm -ti \
+    ${SUDO} podman run --rm -ti \
```

Comment:
Why is this needed?

Reply:
rootfs is owned by root after the RPM extraction. So need to use a rootful container to relabel.